bin/as.sh

```diff
@@ -155,6 +155,8 @@ update_if_necessary()
 {
     local update_version=$1
 
+    echo "${ARTHAS_LIB_DIR}/${update_version}"
```

Comment:
why echo here?

Reply:
@hengyunabc My bad (long time no debugging shell `bash -x test.sh` should work as required). I will remove it.